src/clients/python/__init__.py

```diff
@@ -1481,6 +1412,12 @@ def async_run_with_cb(self, callback, inputs, outputs, batch_size=1, flags=0):
             The batch size of the inference. Each input must provide
             an appropriately sized batch of inputs.
 
+        corr_id : int
+            The correlation id of the inference. If non-zero this
+            correlation ID overrides the context's correlation ID for
+            all subsequent inference requests, else the context retains
```

Comment:
".. else the inference request uses the context's correlation ID"

Reply:
fixed